src/main/kotlin/org/malachite/estella/services/MailService.kt

```diff
@@ -16,116 +17,27 @@ class MailService(
-                I with our team are reviewing your application and will be in touch if we think you’re a potential match for the position.
-                All the best,
-                $hrPartnerFullName
-                """.trimIndent(),
-            sender_name = hrPartnerFullName,
-            sender_email = offer.creator.user.mail
-        )
-    }
-
-    fun getInterviewInvitationAsMailPayload(offer: Offer, interview: Interview): MailPayload {
-        val hrPartnerFullName = "${offer.creator.user.firstName} ${offer.creator.user.lastName}"
-        val url = "${MAIN_URL}interview/${interview.id}"
-        return offer.creator.organization.name.let {
-            MailPayload(
-                subject = "Your are invited for interview with ${it}!",
-                receiver = interview.application.jobSeeker.user.mail,
-                content = """
-                Hi ${interview.application.jobSeeker.user.firstName}
-                Thanks so much for your interest in joining the ${it}! 
-                We are excited to move you forward in our engineering recruiting process.
-                Next step will be interview with our recruiters. It will take place at $url
-                All the best,
-                $hrPartnerFullName
-                """.trimIndent(),
-                sender_name = hrPartnerFullName,
-                sender_email = offer.creator.user.mail
-            )
-
-        }
-    }
-
-    fun organizationVerificationMailPayload(organization: Organization, verified: Boolean) =
-        MailPayload(
-            subject = "Your company has been ${if (verified) "verified" else "unverified"}!",
-            sender_name = "e-Stella Team",
-            receiver = organization.user.mail,
-            content = if (verified) getVerificationText() else getUnVerificationText(),
-            sender_email = MAIN_MAIL
-        )
-
-    fun getVerificationText() =
-        """Your company was successfully verified! You can log in now to your account!"""
+    fun sendRegisterMail(user:User) =
+        sendMail(userRegistrationMailPayload(user))
 
-    fun getUnVerificationText() =
-        """We're sorry to inform you that your company was unverified and so your account was disabled. Please, contact us
-            at estellaagh@gmail.com to resolve this issue.
-        """.trimMargin()
+    fun sendHrPartnerRegisterMail(hrPartner: HrPartner,password:String)=
+        sendMail(hrPartnerRegistrationMailPayload(hrPartner,password))
 
-    fun userRegistrationMailPayload(user: User) =
-        MailPayload(
-            subject = "Thank you for register",
-            sender_name = "e-Stella Team",
-            receiver = user.mail,
-            content = getRegistrationText(),
-            sender_email = MAIN_MAIL
-        )
+    fun sendOrganizationVerificationMail(organization: Organization,verified:Boolean) =
+        sendMail(organizationVerificationMailPayload(organization, verified))
```

Comment:
Tak się w sumie zastanawiam, te wszystkie payloady można by zrobić jako extension functions, tj.
```kotlin
Organization.toVerificationMailPayload(verified) = ...
```
Nie musisz tego robić tutaj, nie wiem, czy to coś bardzo zmienia, ale wydaje mi się trochę lepszym rozwiązaniem, anżeli dokładać parametr ogranization